src/app/accounts/add-api-key-for-account.ts

```diff
@@ -0,0 +1,25 @@
+import { hashApiKey, randomApiKey } from "@domain/accounts"
+import { AccountApiKeysRepository } from "@services/mongoose"
+
+export const addApiKeyForAccount = async ({
+  accountId,
+  label = "",
```

Comment:
IMO defaults shouldn't be set at the use case level. Client should decide what the default should be.